app/build.gradle

```diff
@@ -77,6 +77,10 @@ dependencies {
     runtimeOnly 'io.jsonwebtoken:jjwt-impl:0.11.5'
     runtimeOnly 'io.jsonwebtoken:jjwt-jackson:0.11.5'
 
+    // azure sdk
+    implementation 'com.azure:azure-security-keyvault-secrets:4.5.4'
+    implementation 'com.azure:azure-identity:1.8.0'
```

Comment:
See if this fixes the vulnerability.

```suggestion
    implementation 'com.azure:azure-identity:1.8.1'
```

Reply:
Trying the update. I meant for the Azure one, not the local ;)